src/screens/ExchangeModal.js

```diff
@@ -193,6 +193,7 @@ const ExchangeModal = ({
   const [lastFocusedInput, handleFocus] = useMagicFocus(inputFieldRef.current);
   const [createRefocusInteraction] = useInteraction();
   const isScreenFocused = useIsFocused();
+  const wasScreenFocused = usePrevious(isScreenFocused && !wasScreenFocused);
```

Comment:
the "!wasScreenFocused" part is trying to use a variable that hasn't been declared yet so I think in Javascript land in non-strict mode it'll search for it in global scope, so it will always return !undefined and will always be true, which means it'll just evaluate based on isScreenFocused. You can get rid of the !wasScreenFocused part.

Reply:
I think you were trying to prevent the case where isFocused is always true and that changes wasScreenFocused to true as well, but you're covered bc you check for isScreenFocused && !wasScreenFocused later on, so you're only detecting when there's a change

Reply:
That was the idea